packages/search/src/content/index.ts

```diff
@@ -89,13 +89,195 @@ export class ContentSearchService
  * A standalone function for searching content across the Portal API only or Portal API and the
  * Hub Indexer V3 API.
  *
- * @param request - the IContentSearchRequest instance for searching
+ * The `filter` argument allows the caller to filter results based on attributes exposed by AGO and
```

Comment:
I would insert a short intro before jumping into `filter` and `options`. For example
> A content search is configured by passing `searchContent` an object of type `IContentSearchRequest`. This configuration object is composed of two important child-objects, `filter` and `options`.